src/DotvvmAcademy.Web/Pages/site.dotmaster

```diff
@@ -6,11 +6,23 @@
       lang="{resource: LanguageMoniker}"
       RenderSettings.Mode="Server">
 <head>
+    <script>
+        (function (i, s, o, g, r, a, m) {
+            i['GoogleAnalyticsObject'] = r; i[r] = i[r] || function () {
+                (i[r].q = i[r].q || []).push(arguments);
+            }, i[r].l = 1 * new Date(); a = s.createElement(o),
+                m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m);
+        })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
+
+        ga('create', 'UA-68593658-2', 'auto');
+        ga('send', 'pageview');
+    </script>
```

Comment:
Why don't we use https://github.com/riganti/dotvvm-contrib/blob/master/Controls/GoogleAnalyticsJavascript/readme.md?